community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/CompatibilityFor2_3.scala

```diff
@@ -195,17 +195,17 @@ case class ExecutionResultWrapperFor2_3(inner: InternalExecutionResult, planner:
     new ResourceIterator[util.Map[String, Any]] {
       def close() = exceptionHandlerFor2_3.runSafely {
         endQueryExecution()
-        innerJavaIterator.close
+        innerJavaIterator.close()
```

Comment:
It is not needed per se, but if you declare a def with parenthesis you should also call it using parenthesis, it works without but you get compiler warnings.